src/main/java/mate/jdbc/util/ConnectionUtil.java

```diff
@@ -0,0 +1,27 @@
+package mate.jdbc.util;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    static {
+        try {
+            Class.forName("com.mysql.cj.jdbc.Driver");
+        } catch (ClassNotFoundException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    public static Connection getConnection() {
+        try {
+            Properties dbProperties = new Properties();
+            dbProperties.put("user", "root");
+            dbProperties.put("password", "1234567");
+            return DriverManager.getConnection("jdbc:mysql://localhost:3306/taxi_db", dbProperties);
+        } catch (SQLException e) {
+            throw new RuntimeException("Can't create connection to db", e);
```

Comment:
Exception handling: Exception handling could be improved. Instead of throwing a RuntimeException right away, it's better to log errors and error messages or handle them more carefully.